src/docfx.website.themes/common/common.js

```diff
@@ -118,3 +119,30 @@ function getOverrideTemplate(uid) {
     content += "\n";
     return content;
 }
+
+function mergeSeeAlso(model) {
+    var seealso = [];
```

Comment:
addIsCref() already adds the key `isCref` to it. No need to construct a new list?